professor.rb

```diff
@@ -0,0 +1,34 @@
+require_relative 'person'
+
+class Professor < Person
+  attr_reader :salary, :phase, :target_raise
+
+  def initialize(options={})
+    @phase = 3  
+    @age = options.fetch(:age, 0)
+
+    @name = options.fetch(:name, "")
+  end
+
+  def set_phase(num)
+    @phase = num
+    "Cool, I've always wanted to teach phase #{num}!"
+  end
+
+  def teach_stuff
```

Comment:
this method also appears in the ApprenticeTeacher class. How can this be DRY'd up?